### PR TITLE
Update openshift ruleset documentation

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/WINDUPRULE-646

Ready for merge

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

File: rules-reviewed/openshift/session.windup.xml [hunk 1]
```diff
@@ -26,37 +26,28 @@
...
-                       Each node in the cluster shares information about ongoing sessions and can take over sessions if a node disappears.
-                       The `<distributable/>` tag inside the `<web-app>` tag of application’s `web.xml` descriptor file enables the application's sessions clustering [1].
+                    Session replication ensures that client sessions are not disrupted by node failure. Each node in the cluster shares information about ongoing sessions and can take over sessions if another node disappears. In a cloud environment, however, data in the memory of a running container can be wiped out by a restart.
 
-                       In a cloud environment it has to be considered that the data in the memory of a running container can be wiped out by a restart (triggered by code deploy, config change, or the execution environment relocating the process to a different physical location)[2].
+                    Recommendations:
 
-                       Different approaches can be followed:
+                    Review the session replication usage and ensure that it is configured properly.
```
Comment (jonathanvila) [inline rules-reviewed/openshift/session.windup.xml:33]: if the section is "Recommendations:" , wdyt on having lines 33, 35,27 as an ordered list or bullet list ?

File: rules-reviewed/openshift/session.windup.xml [hunk 2]
```diff
@@ -72,29 +63,22 @@
...
 
-                    In a cloud environment it has to be considered that the data in the memory of a running container can be wiped out by a restart (triggered by code deploy, config change, or the execution environment relocating the process to a different physical location)[1].
-                    Consider storing HTTPSession data to a cache backing service[2][3][4][5]
-                    This approach has some benefits:
+                    Recommendations:
 
-                    * Increased application scalability and elasticity. By offloading the session data off to a remote Data Grid, the application tier itself can be more scalable and elastic.
-                    * Session Persistence. By offloading session data to a remote data grid, the application itself will be able to survive EAP node failures since the a JVM failure will not cause the session data to be lost.
-                    * Session Data Sharing. If you have a requirement for multiple applications to be able to share session data, this solution might be able to solve that use case as well.
+                    Store HTTP session data in a cache backing service or a remote data grid.
```
Comment (jonathanvila) [inline rules-reviewed/openshift/session.windup.xml:70]: in this case there's only 1 recommendation .... does it still make sense to have the heading "Recommentations:" ?

File: rules-reviewed/openshift/session.windup.xml (hunk 1 above)
Comment (apinnick) [reply rules-reviewed/openshift/session.windup.xml:33]: Agree

File: rules-reviewed/openshift/session.windup.xml (hunk 2 above)
Comment (apinnick) [reply rules-reviewed/openshift/session.windup.xml:70]: It's like "Prerequisites". We use that title even if there is only ONE prerequisite. But doing that in a ruleset might be overkill. I'll check.

**Review by jonathanvila — APPROVED** [review]
LGTM :+1:

**Comment by apinnick** [comment]
@PhilipCattanach Ready to merge

**Pull request merged**